pkg/controller.v1beta1/suggestion/suggestionclient/suggestionclient_test.go

```diff
@@ -539,6 +539,138 @@ func TestConvertParameterType(t *testing.T) {
 	}
 }
 
+func TestConvertFeasibleSpace(t *testing.T) {
+
+	tcs := []struct {
+		inFeasibleSpace       experimentsv1beta1.FeasibleSpace
+		expectedFeasibleSpace *suggestionapi.FeasibleSpace
+		testDescription       string
+	}{
+		{
```

Comment:
I don't think that all 3 cases should be useful.
So, could you add only 2 cases, "distribution is Distribution_DISTRIBUTION_UNKNOWN", "distribution is not Distribution_DISTRIBUTION_UNKNOWN"?

Reply:
Yes, removed the extra test cases.